### ISSUE TITLE
Add the ability to always show RichTextKeyboardToolbarStyle

### ISSUE THREAD

**Issue by r-jarvis** [danielsaidi/RichTextKit#115]
Hello!

I was browsing through some rich text libraries and was hoping to use this on one of my projects! One strict requirement we have is that the editing toolbar is always shown, not only when the text editor is selected. Looking through the code I did not see this implemented anywhere (Let me know if I just missed this functionality somewhere).

Is this feature something that you think would benefit the library? If so, I'd be willing to code it up and create a PR. Feel free to close this ticket if you aren't interested in this feature.

**Comment by danielsaidi** [comment]
Hi @r-jarvis 

Sure, these are just regular SwiftUI views, so you can just add any toolbar or sheet to any part of the screen as you like :)

**Comment by DominikBucher12** [comment]
I have somewhere demo with those, we can maybe create Demo with different toolbar settings?

**Comment by danielsaidi** [comment]
That would be amazing.

I'm planning to rebuild the demo from scratch before we release 1.0.

**Comment by r-jarvis** [comment]
Ah I thought that since opacity/offest/frame are based on `context.isEditingText`, you would need to make sure those values are set correctly as well.

https://github.com/danielsaidi/RichTextKit/blob/b41066746f7ba00bdc15582268489818bd21a4fe/Sources/RichTextKit/Keyboard/RichTextKeyboardToolbar.swift#L136

I threw together this small change that adds in a settings for `RichTextKeyboardToolbarStyle` so that you could initialize with it and have it always displayed. This seemed like a fairly low-cost solution and has it easily managed by the user in `RichTextKeyboardToolbarStyle.init()`.

https://github.com/r-jarvis/RichTextKit/commit/5a07584507ef080943e507114134db5e91d9b276

I just started looking over the project so I'm sure I'm missing something here :)

**Comment by danielsaidi** [comment]
That's great Ryan!

I don't think the configuration belongs in the style, but think that every component should have a style and a configuration. The configuration could contain that logic.

**Comment by r-jarvis** [comment]
Yeah I definitely agree with that. I threw it in there for now since I wasn't sure if you wanted each component to have a configuration (since, at least for now, would only contain this one setting). I can make that change and add in a `RichTextKeyboardToolbar` config. I'll make a PR when it's done.

Thanks for the feedback!
